node/resolvers/orderForm.ts

```diff
@@ -34,6 +34,16 @@ export const root = {
   OrderForm: {
     id: prop('orderFormId'),
     marketingData: propOr({}, 'marketingData'),
+    allowManualPrice: async (
+      orderForm: CheckoutOrderForm,
+      _: unknown,
+      ctx: Context
+    ) => {
+      return (
+        (await ctx.clients.checkoutAdmin.orderForm())?.allowManualPrice ||
```

Comment:
I would create a separate line for this.

Reply:
sure!